src/main/java/org/prebid/server/bidder/appnexus/AppnexusBidder.java

```diff
@@ -76,7 +76,7 @@ public class AppnexusBidder implements Bidder<BidRequest> {
     private static final String POD_SEPARATOR = "_";
     private static final int MAX_IMP_PER_REQUEST = 10;
 
-    private static final TypeReference<ExtPrebid<?, ExtImpAppnexus>> APPNEXUS_EXT_TYPE_REFERENCE =
+    private static final TypeReference<ExtImp> APPNEXUS_EXT_TYPE_REFERENCE =
             new TypeReference<>() {
             };
```

Comment:
With this change, there is no need in `TypeReference`. Just use `.convertValue(...)` providing `AppnexusExtImp.class`